js-developer/README.md

```diff
@@ -8,6 +8,9 @@ Build a [Single Page Application](https://en.wikipedia.org/wiki/Single-page_appl
 1. There should be another view to add, delete and edit contacts.
 1. The data should be persisted (on the client), and loaded again when the application starts.
 1. Add fitting validation to the different input fields.
+1. Add support for multiple languages.
```

Comment:
Surely it'd be good to check how candidate approaches such challenge though I think the task is already complex enough and needs an investment of a big portion of candidate's time. E.g. for Yago it took one week to deliver a good quality solution. Perhaps we can put the internationalisation instead of something else..?

Reply:
It definitely adds more complexity but I think it is a feature that requires the candidate to really plan carefully for architecture and will expose more his decision making and architectural knowledge (which I'm lacking a bit to see at the moment).

Maybe we can remove the "Delete contact" and the "Country list" features? The latest I believe was added if the candidate was familiar with modules but I think at this stage it is really easy to see.

Reply:
I think Edit & Delete options are basic functionality of an address book, they should stay as requirements (actually some people failed those, from my experience). How about making it optional to senior devs? Or keep it as an option for a later stage - we could ask candidate in regards to his assignment about how he / she would implement internationalisation?

Reply:
"Optional" sections tend to be regarded as mandatory in any case. Ok, let's scratch this from the initial requirements and keep it as an option for further stages if required